### PR TITLE
Removed aria-checked attribute, $ sign from alert message

### PR DESCRIPTION
**Pull request by pavi41**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.frontend/src/LanguageUtils.js [hunk 1]
```diff
@@ -94,6 +94,9 @@ class LanguageUtils {
                         ? snippets[number]
                         : match;
                 });
+
+                //  Remove all the $ symbols in alet message
+                     translatedText = translatedText .replace(/\$/g, '');
```
Comment (rismehta) [inline ui.frontend/src/LanguageUtils.js:99]: Can we change the main regex to `/\${(\d+)}/g`

Comment (pavi41) [reply ui.frontend/src/LanguageUtils.js]: Done!

**Review by rismehta — CHANGES_REQUESTED** [review]
Please write test cases for this change

**Comment by pavi41** [comment]
> Please write test cases for this change

Done!

**Review by rismehta — CHANGES_REQUESTED** [review]
Please also add test cases for the alert box which is shown in file attachment. 

if its not possible with the current cypress version, you can wait for this PR, https://github.com/adobe/aem-core-forms-components/pull/1353, to be merged.

**Comment by rismehta** [comment]
The spec.js file needs to renamed to cy.js file

**Pull request merged**